JellyfishClientSdk/Package.swift

```diff
@@ -5,24 +5,29 @@ import PackageDescription
 
```

Comment:
Added readme, however I think it's not that relevant right now since after this PR is merged I'd have to redo it since the project structure will change to mimic the webrtc repo